### DISCUSSION TITLE
Will he keep Torch if i put on?

### DISCUSSION THREAD

**Discussion by tyson-dev** [blizzhackers/kolbot-SoloPlay#68] [Q&A about kolbot SoloPlay]
Will he keep Torch if i put on Hammerdin currently leveling?

**Comment by aim2kill** [comment]
He should

**Comment by tyson-dev** [comment]
Ty aim,

it works.
He will not self euquip (keeps in stash) but when put in inv he keeps it there and even does sort it to the top right.

cheers
